Week3/strings-arrays.js

```diff
@@ -0,0 +1,37 @@
+//1.Strings
+let myString = "hello,this,is,a,difficult,to,read,sentence";
+console.log(myString);
+myString=myString.replace(/,/g, " "); // that's totally google answer i couldn't find what the g stands for
```

Comment:
Ah, what the `replace` does is, replace the first `,` in your string with a ` `. And when you add the `/g` it will replace every `,` to ` `. So it will give the right result. 

Did this help? Or is it still confusing?